examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -63,9 +68,23 @@ impl FrontendNode {
             backend_in.push(read_handle);
         }
 
+        // Build a unique storage name, so different test runs don't affect each other.
+        let mut storage_name = STORAGE_NAME_PREFIX.to_vec();
+        let mut nonce = [0; 16];
+        let _ = oak::random_get(&mut nonce);
+        let hex_nonce = hex::encode(nonce);
+        storage_name.append(&mut hex_nonce.into_bytes());
```

Comment:
I think something like this may be clearer:

```rust
let storage_name = format!("{}-{}", STORAGE_NAME_PREFIX, hex_nonce);
```

And then call `as_bytes` or `into_bytes` if / where necessary.